libnavigation-ui/src/main/java/com/mapbox/navigation/ui/voice/MapboxSpeechPlayer.java

```diff
@@ -160,17 +162,28 @@ private void onError(String errorText) {
   }
 
   private void playInstruction(@NonNull File instruction) {
-    setupMediaPlayer(instruction.getPath());
+    setupMediaPlayer(instruction);
   }
 
-  private void setupMediaPlayer(String instructionPath) {
-    if (TextUtils.isEmpty(instructionPath)) {
+  private void setupMediaPlayer(@NonNull File instruction) {
+    if (!instruction.canRead()) {
+      String msg = "Cannot read file " + instruction.getAbsolutePath();
```

Comment:
Extract this magic number into a constant?

Reply:
Not really a magic number, is it? String speaks for iteslf.

Reply:
Debatable 👍